src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/VisualRuleForm.tsx

```diff
@@ -0,0 +1,238 @@
+                                config,
+                                sceneId,
+                                selectedElements
+                            }}
+                            mode={ModelledPropertyBuilderMode.TOGGLE}
+                            propertyExpression={{
+                                expression: ''
+                            }}
+                            onChange={onPropertyChange}
+                            onInternalModeChanged={onInternalModeChanged}
+                            required
+                        />
+                        {isPropertyTypeDropdownEnabled && (
+                            <Dropdown
+                                required
+                                placeholder={t(
+                                    '3dSceneBuilder.visualRuleForm.typePlaceholder'
+                                )}
+                                label={t('type')}
+                                selectedKey={`value-type-${formData.propertyType}`}
+                                onChange={onTypeChange}
+                                options={typeOptions}
+                                onRenderOption={onRenderTypeOption}
+                                onRenderTitle={onRenderTypeTitle}
+                                styles={classNames.subComponentStyles.dropdown}
+                            />
+                        )}
+                    </Stack>
+                </div>
+                <Separator />
+                <div className={commonFormStyles.expandingSection}>
+                    <Stack horizontal verticalAlign={'center'}>
+                        <Label styles={classNames.subComponentStyles.label}>
+                            {t('3dSceneBuilder.conditions')}
+                        </Label>
+                        <TooltipCallout
+                            content={{
+                                buttonAriaLabel: t(
+                                    '3dSceneBuilder.visualRuleForm.conditionsInfoContent'
+                                ),
+                                calloutContent: t(
+                                    '3dSceneBuilder.visualRuleForm.conditionsInfoContent'
+                                )
+                            }}
+                            styles={
+                                classNames.subComponentStyles.tooltipCallout
+                            }
+                        />
+                    </Stack>
+                    <ConditionsList
+                        styles={classNames.subComponentStyles.conditionsList}
+                    />
+                </div>
+            </div>
+            <PanelFooter>
+                <PrimaryButton
+                    text={t('save')}
+                    onClick={() =>
+                        setVisualRuleFormMode(VisualRuleFormMode.Inactive)
+                    }
+                    styles={
+                        classNames.subComponentStyles.saveButton
```

Comment:
nit: no need for if/else if it is set to `undefined` in else
```suggestion 
                        classNames.subComponentStyles?.saveButton
```